infrastructure/main.tf

```diff
@@ -21,6 +21,20 @@ module "blob-router-db" {
   subscription       = "${var.subscription}"
 }
 
+module "reform-blob-router-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = "${var.product}-${var.component}"
+  location           = "${var.location_db}"
+  env                = "${var.env}"
+  database_name      = "reform_blob_router"
```

Comment:
should it not be `blob_router`? it's the name of the schema/database

Reply:
we already have that name